integration_tests/base_routes.py

```diff
@@ -606,6 +603,15 @@ async def request_json(request: Request):
     return json["key"]
 
 
+@app.post("/sync/request_json/json_type")
+async def request_json_type(request: Request):
+    try:
+        jsondict = request.json()
+    except ValueError:
+        jsondict = None
+    return jsonify(jsondict)
```

Comment:
you can just return a dict here. No need to use jsonify

Reply:
I try just return dict .  but it error .   dict just encode by python ,  
return this dict :
```
dc= [{'k': 'v'}, {'k': None}]

```
it response.body is :
```
[{'k': 'v'}, {'k': None}]
```
This is not a valid JSON string. it is a string directly serialized from a dictionary. 

so , I used  jsonify